analysis.json

```diff
@@ -2,7 +2,7 @@
   "schema_version": "1.0.0",
   "elements": [
     {
-      "description": "`<cosmoz-theme-switcher>` is a custom style theme manager\n\n### Usage\n\n    <cosmoz-theme-switcher></cosmoz-theme-switcher>",
+      "description": "`<cosmoz-theme-switcher>` is a custom style theme manager\n\n### Usage\n\n    <cosmoz-theme-switcher></cosmoz-theme-switcher>\n\neslint no-unused-vars: [\"warn\", { \"argsIgnorePattern\": \"event\" }]",
```

Comment:
Hmm, where does this "eslint no-unused-vars" comes from at the end of the description ?

Will it be displayed somewhere ?
I've checked http://127.0.0.1:8081/components/cosmoz-theme-switcher/#/elements/cosmoz-theme-switcher and don't see it.

Reply:
two functions(themeError, themeLoaded) had unused param `event` I have removed that and generated a new analysis.json. added also arrow functions where needed and replaced var with const;